.github/workflows/tests.yml

```diff
@@ -288,7 +288,7 @@ jobs:
     # TODO: when actions supports using ${{env}} in job.*.if and not just job.*.steps.*.if, use this:
     #if: ${{ env.NIGHTLY }}
     # in the meantime, copy-paste this:
-    if: ${{ github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
+    # if: ${{ github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' }}
```

Comment:
Hrm. That didn't quite go according to plan. I expected that uncommenting the `if` statements would cause the WSL runners to be triggered, but that didn't work here.

I suppose I can try manually triggering the run? :thinking:

Reply:
I manually ran the tests for both:

- `windows-2019` + `setup-wsl@v3`
- `windows-2022` + `setup-wsl@v2`

The former passes but the latter fails, so it looks like `windows-2022` is the culprit. 

At least this give us a very straightforward path to fixing out CI in the short-term. Though, at some point we'll need to think about how to support WSL testing if/when GH deprecates `windows-2019`.